backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/document/model/dsl/impl/ElasticsearchIndexSchemaFieldContextImpl.java

```diff
@@ -7,8 +7,6 @@
 package org.hibernate.search.v6poc.backend.elasticsearch.document.model.dsl.impl;
 
 import java.time.LocalDate;
-import java.util.Arrays;
-import java.util.List;
```

Comment:
commit message: s/an manipulate lists/and manipulate lists/

Reply:
Done